mbed-util/FunctionPointerBase.h

```diff
@@ -38,6 +46,7 @@ class FunctionPointerBase {
     virtual void clear() {
         _membercaller = NULL;
         _object = NULL;
+        memset(_member, 0, sizeof(_member));
```

Comment:
A private function that sets the member to 0 would be just as readable and more efficient.